src/retasc/__main__.py

```diff
@@ -45,6 +46,14 @@ def parse_args():
         action="store_true",
     )
 
+    subparsers.add_parser(
+        "run", help="Process rules, data from Product Pages and apply changes to Jira"
+    )
+    subparsers.add_parser(
```

Comment:
I'm not yet sure how useful it would be. :thinking: Actually, I need to add a `dry_run` property also to `RuntimeContext` so any custom prerequisites could check if they can apply changes if needed. But it is also easy to forget there is a dry-run mode.

I think it is good to keep it for testing/trouble-shooting against a real Jira server.